Storage/Storage/Tools/StorageType+Extensions.swift

```diff
@@ -35,6 +35,14 @@ public extension StorageType {
 
     // MARK: - Orders
 
+    /// Retrieves the Stored Orders given the IDs.
+    ///
+    func loadOrders(siteID: Int64, orderIDs: [Int64]) -> [Order] {
+        let predicate = NSPredicate(format: "orderID in %@", orderIDs)
```

Comment:
Do we want to add the `siteID` to the predicate? With the current predicate, I am afraid that we will end up loading orders from other sites with the same order ID.

Reply:
Good catch! I updated the predicate in a27533c7133933d91e0cf80da4a3e24259c73ddd.